diagnostic.md

```diff
@@ -54,7 +68,10 @@ How would we reference each of the following categories of elements?
 -   All `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
-
+.big
+#contentPane
+.important .red
+div.children
```

Comment:
`div > p` would be the correct way. Notice above, the space is a descendent relationship but the > acts a parent/child relationship. They are different (Descendent vs Child)